### PR TITLE
Change: Code refactor

### PR DESCRIPTION
**Pull request by amrit-choudhary**
Changes: Part 1 of code refactor. Moved files to appropriate folders and corrected references. Now has a better folder structure. Part 2 will involve actual code refactor, reducing global vars etc.

### CODE REVIEW

File: game/screens/pause_screen.py [hunk 1]
```diff
@@ -10,16 +10,8 @@ def import_modify():
...
-    from .start_screen import *
-    from .global_funcs import *
-    from .global_objects import *
-    from .constants import *
-except SystemError as er:
-    from start_screen import *
-    from global_funcs import *
-    from global_objects import *
-    from constants import *
+from .start_screen import *
```
Comment (thealphadollar) [inline game/screens/pause_screen.py]: Why have you removed absolute importing?

**Comment by thealphadollar** [comment]
Please do since imports are dependent on the way the game is launched.

**Comment by thealphadollar** [comment]
Great work, have you tested this thoroughly?

**Comment by amrit-choudhary** [comment]
Yes, but you should give it a try as well.

**Comment by thealphadollar** [comment]
Sorry for the delay, I'm sick.

**Comment by amrit-choudhary** [comment]
@thealphadollar  no issues; get well soon.

**Pull request merged**